iron-form.html

```diff
@@ -88,25 +111,45 @@
 
     /**
      * Returns a json object containing name/value pairs for all the registered
-     * custom components and native elements of the form.
+     * custom components and native elements of the form. If there are elements
+     * with duplicate names, then their values will get aggregated into an
+     * array of values. 
      */
     serialize: function() {
       var json = {};
 
+      function addSerializedElement(el) {
+        // If the name doesn't exist, add it. Otherwise, serialize it to
+        // an array,
+        if (!json[el.name]) {
+          json[el.name] = el.value;
+        } else {
+          if (typeof(json[el.name]) !== 'object') {
```

Comment:
Oh I didn't know there was a thing. Thank god, that `typeof` was giving me the creeps.